index.html

```diff
@@ -0,0 +1,412 @@
+                <a href="#">About</a>
+            </li>
+            <li>
+                <a href="#">Sample page</a>
+            </li>
+            <li>
+                <a href="#">Portfolio</a>
+            </li>
+            <li>
+                <a href="#">Blog</a>
+            </li>
+            <li class="dropdown">
+                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parent page
+                    <span class="caret"></span>
+                </a>
+                <ul class="dropdown-menu">
+                    <li>
+                        <a href="#">Full page</a>
+                    </li>
+                    <li>
+                        <a href="#">Child page 1</a>
+                    </li>
+                    <li>
+                        <a href="#">Child page 2</a>
+                    </li>
+                </ul>
+            </li>
+            <li>
+                <a href="#">Contact</a>
+            </li>
+        </ul>
+    </nav>
+
+    <section>
+
+        <div id="carousel-example-generic" class="carousel slide" data-ride="carousel">
+            <!-- Indicators -->
+            <!-- <ol class="carousel-indicators">
+            <li data-target="#carousel-example-generic" data-slide-to="0" class="active"></li>
+            <li data-target="#carousel-example-generic" data-slide-to="1"></li>
+            <li data-target="#carousel-example-generic" data-slide-to="2"></li>
+            <li data-target="#carousel-example-generic" data-slide-to="3"></li>
+
+        </ol> -->
+
+            <!-- Wrapper for slides -->
+            <div class="carousel-inner" role="listbox">
+                <div class="item active">
+                    <img src="starter-code/img/slider-1.jpg" alt="slider-1">
+                    <div class="carousel-caption">
+                        <div id="slider">
+                            <h2>ETIAM MAURIS TORTOR, PHARETRA QUIS LOBORTIS IN, PHARETRA IN DIAM</h2>
+                            <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed
+                                metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nec.
+                                Suspendisse at erat luctus; hendrerit sapien sed, consectetur erat. Maecenas dignissim suscipit
+                                orci at molestie. Aenean fringilla dolor vitae lacus lacinia eleifend. Suspendisse et libero
+                                nunc. Etiam mauris tortor, pharetra […]</p>
+                            <button type="button" class="btn btn-default">READ MORE</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="item">
+                    <img src="starter-code/img/slider-2.jpg" alt="slider-2">
+                    <div class="carousel-caption">
+                        <div id="slider">
```

Comment:
Don't use the same id for more than one element, it defeats the semantic purpose of an id as being a unique way to select an element. Use a different id or use a class.